src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -360,15 +353,12 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
                   logger.info(s"$userStr download via proxy allowed for [$objectStr]")
                   val gcsApiUrl = getXmlApiMetadataUrl(bucketName, objectKey)
                   val extReq = Get(gcsApiUrl)
-                  val proxyPipeline = addCredentials(userAuthToken.accessToken) ~> sendReceive
-                  // ensure we set the content-type correctly when proxying
-                  proxyPipeline(extReq) map { proxyResponse:HttpResponse =>
-                      proxyResponse.header[HttpHeaders.`Content-Type`] match {
+
+                  executeRequestRaw(userAuthToken.accessToken)(extReq).map { proxyResponse =>
+                    proxyResponse.header[`Content-Type`] match {
                       case Some(ct) =>
-                        RequestCompleteWithHeaders((proxyResponse.status, proxyResponse.entity),
-                          HttpHeaders.`Content-Type`(ct.contentType)
-                        )
-                      case None => RequestComplete((proxyResponse.status, proxyResponse.entity))
+                        RequestCompleteWithHeaders(proxyResponse, `Content-Type`(ct.contentType))
+                      case None => RequestComplete(proxyResponse)
```

Comment:
i think that this and the above line are equivalent to their previous implementations but have not tested